admin/test/test_release.py

```diff
@@ -1412,3 +1418,164 @@ def get_dispatcher(self):
         self.assertTrue(
             expected_files.issubset(set(files_on_s3)),
             "Metadata files for the packages were not created.")
+
+
+class CreateReleaseBranchOptionsTests(TestCase):
+    """
+    Tests for :class:`CreateReleaseBranchOptions`.
+    """
+
+    def test_flocker_version_required(self):
+          """
+          The ``--flocker-version`` option is required.
+          """
+          options = CreateReleaseBranchOptions()
+          self.assertRaises(
+              UsageError,
+              options.parseOptions, [])
+
+class CreateReleaseBranchTests(TestCase):
+    """
+    Tests for :func:`create_release_branch`.
+    """
+
+    def setUp(self):
+        repo_directory = FilePath(tempfile.mkdtemp())
```

Comment:
If you use Twisted's `TestCase` or `SynchronousTestCase` you can use `self.mktemp()` to create temporary file paths. They end up in `_trial_temp` in test-specific directories. This means when debugging failing tests it's easy to figure out which files/directories belong to which test. It also means you don't need (or want to) cleanup after creating them unless maybe you created a really huge file.